configs/dictionaries/active_ab_tests.yaml

```diff
@@ -7,4 +7,4 @@ Example: true
 ViewDrivingLicence: false
 FinderAnswerABTest: true
 SearchClusterQueryABTest: true
-LandingPageTest: false
+LandingPageTest: true
```

Comment:
Should we name this as BrexitLandingPageTest as I'm not sure LandingPage makes this sufficiently distinct